notifications/11-21-22-1530.md

```diff
@@ -0,0 +1,30 @@
+**What is happening?**
+
+We are going to enable http compression on the ingress controller for the PROD clusters. This has been tested in the LAB successfully. It should reduce network usage and improve page loading time for users. We do not anticipate any outages or service disruptions as the ingress pods will cycle one at a time to apply the change.
```

Comment:
```suggestion
We are going to enable HTTP compression on the ingress controller for the PROD clusters. This has been tested in the LAB successfully. It should reduce network usage and improve page loading time for users. We do not anticipate any outages or service disruptions as the ingress pods will cycle one at a time to apply the change.
```

Reply:
We should also include the mime types that will be compressed, that only already uncompressed resources will be affected, and that passthrough TLS routes will be unaffected.